resources/README.md

```diff
@@ -1,6 +1,6 @@
 # Sippy Sample OpenShift Deployment
 
-This directory contains a sample deployment of Sippy on OpenShift, configured roughly as needed to pull job data from OpenShift CI's Prow instance.
+This directory contains a **sample** deployment of Sippy on OpenShift, configured roughly as needed to pull job data from OpenShift CI's Prow instance (Production sippy deployment is managed in [continuous-release-jobs.](https://github.com/openshift/continuous-release-jobs/tree/master/config/clusters/dpcr/services/sippy)).
```

Comment:
This repo is private so I'm not entirely sure we want to link to it.

Reply:
Fair enough, it was just a breadcrumb for myself really.